packages/ui/src/components/Table/STable.vue

```diff
@@ -0,0 +1,819 @@
+    /**
+     * Whether width of column automatically fits its container
+     * */
+    fit?: boolean
+    /**
+     * Whether table header is visible
+     * */
+    showHeader?: boolean
+    /**
+     * Whether current row is highlighted
+     * */
+    highlightCurrentRow?: boolean
+    /**
+     * Key of current row, a set only prop
+     * */
+    currentRowKey?: string | number
+    /**
+     * Function that returns custom class names for a row, or a string assigning class names for every row
+     * */
+    rowClassName?: string | ((param: TableRowConfigCallbackParams) => string)
+    /**
+     * Function that returns custom style for a row, or an object assigning custom style for every row
+     * */
+    rowStyle?: CSSProperties | ((param: TableRowConfigCallbackParams) => CSSProperties)
+    /**
+     * Function that returns custom class names for a cell, or a string assigning class names for every cell
+     * */
+    cellClassName?: string | ((param: TableCellConfigCallbackParams) => string)
+    /**
+     * Function that returns custom style for a cell, or an object assigning custom style for every cell
+     * */
+    cellStyle?: CSSProperties | ((param: TableCellConfigCallbackParams) => CSSProperties)
+    /**
+     * Function that returns custom class names for a row in table header, or a string assigning class names for every row in table header
+     * */
+    headerRowClassName?: string | (() => string)
+    /**
+     * Function that returns custom style for a row in table header, or an object assigning custom style for every row in table header
+     * */
+    headerRowStyle?: CSSProperties | (() => CSSProperties)
+    /**
+     * Function that returns custom class names for a cell in table header, or a string assigning class names for every cell in table header
+     * */
+    headerCellClassName?: string | ((param: TableHeaderCellConfigCallbackParams) => string)
+    /**
+     * Function that returns custom style for a cell in table header, or an object assigning custom style for every cell in table header
+     * */
+    headerCellStyle?: CSSProperties | ((param: TableHeaderCellConfigCallbackParams) => CSSProperties)
+    /**
+     * key of row data, used for optimizing rendering. Required if reserve-selection is on.
+     * When its type is String, multi-level access is supported, e.g. user.info.id,
+     * but user.info[0].id is not supported, in which case Function should be used.
+     * */
+    rowKey?: string | ((row: TableRow) => unknown) | null
+    /**
+     * Displayed text when data is empty. You can customize this area with `slot="empty"`
+     * */
+    emptyText?: string
+    /**
+     * whether expand all rows by default,
+     * works when the table has a column type="expand" or contains tree structure data
+     * */
+    defaultExpandAll?: boolean
+    /**
+     * Set expanded rows by this prop. Prop's value is the keys of expand rows,
+     * you should set row-key before using this prop
```

Comment:
```suggestion
     * Which rows to expand. Keys passed into this prop are the ones that
     * are computed by the `row-key` prop.
```

Also, how it works? It expands them "by default", like `default-expand-all`?

Reply:
Yes, but this prop watched